RecoMET/METFilters/python/BadPFMuonFilter_Dz_cfi.py

```diff
@@ -0,0 +1,19 @@
+import FWCore.ParameterSet.Config as cms
+
+BadPFMuonFilter = cms.EDFilter(
```

Comment:
nominally, python module instances should start from lower case, but since other instances are present for BadParticleFilter, these better be moved/renamed at the same time. Can be in a separate PR.

Reply:
II can do it separately later. need to discuss with conveners